fuel-core/src/executor.rs

```diff
@@ -10,17 +10,22 @@ use crate::{
 use fuel_asm::Word;
 use fuel_storage::Storage;
 use fuel_tx::{Address, Bytes32, Color, Input, Output, Receipt, Transaction, UtxoId};
-use fuel_vm::prelude::{Interpreter, InterpreterError};
+use fuel_vm::{
+    consts::REG_SP,
+    prelude::{Backtrace as FuelBacktrace, InterpreterError},
+    transactor::Transactor,
+};
 use std::error::Error as StdError;
 use std::ops::DerefMut;
 use thiserror::Error;
+use tracing::warn;
 
 pub struct Executor {
     pub database: Database,
 }
 
 impl Executor {
-    pub async fn execute(&self, block: &FuelBlock) -> Result<(), Error> {
+    pub async fn execute(&self, block: &FuelBlock, with_backtrace: bool) -> Result<(), Error> {
```

Comment:
It might be cleaner if we cloned the `VMConfig` struct to the executor, since this will apply to all transactions. This pattern of adding config params to the API methods may not scale as the VMConfig gets more options.

Reply:
sure